CC-BY-4.0.txt

```diff
@@ -393,4 +393,3 @@ the avoidance of doubt, this paragraph does not form part of the
 public licenses.
 
 Creative Commons may be contacted at creativecommons.org.
-
```

Comment:
Do not modify the license text files, those come as-is from elsewhere.  If these need to be ignored by the pre-commit hooks, then we should do that.